tailwind.config.js

```diff
@@ -1,8 +1,12 @@
 const colors = require("tailwindcss/colors");
 
 module.exports = {
```

Comment:
_Some_ of the values specified under the `theme.extend.spacing` setting now seem to be covered by [the default spacing scale](https://tailwindcss.com/docs/customizing-spacing#default-spacing-scale) - the repetition here should probably be cleaned up.